skipper.go

```diff
@@ -325,14 +325,14 @@ type Options struct {
 	// DisabledFilters is a list of filters unavailable for use
 	DisabledFilters []string
 
-	// CloneRoute is a PreProcessor, that will be applied to all routes automatically. It
-	// will clone all matching routes and apply changes to the
+	// CloneRoute is a slice of PreProcessor's that will be applied to all routes
+	// automatically. They will clone all matching routes and apply changes to the
 	// cloned routes.
-	CloneRoute *eskip.Clone
+	CloneRoute []*eskip.Clone
 
 	// EditRoute will be applied to all routes automatically and
 	// will apply changes to all matching routes.
-	EditRoute *eskip.Editor
+	EditRoute []*eskip.Editor
```

Comment:
These changes are actually breaking changes.
@AlexanderYastrebov wdyt?

Reply:
We can have new option `EditRoutes` and just append EditRoute to EditRoutes

Reply:
I don't see the case why this could break something.
In both cases I see:
```
./bin/skipper ... -edit-route ... -edit-route ...
```
and
```
edit-route: ... change1 ...
edit-route: ... change2 ...
```
in `config.yaml` both `edit-route` options are working fine.

In case when there is only one `edit-route` options (all use cases at the moment) this only `edit-route` option is still working good.

Reply:
yes binary users are not impaired, but library users will see a change from a single type to a slice type

Reply:
@szuecs , how many library users do we have? Will they have problems only during compile time or during runtime too? It seems to me that if there are not too many users and there are only problems during compile time than it is better force them to recompile one more time but not to add additional workaround field in `skipper.Options` struct.